### ISSUE TITLE
Semantic segmentation logit fusion

### ISSUE THREAD

**Issue by alexgkendall** [mapillary/inplace_abn#30]
Awesome work and thanks for releasing the pretrained weights - when evaluating semantic segmentation do you observe best performance with mean, voting or max fusion of multiple segmentation logits?

**Comment by rotabulo** [comment]
I close the issue.
